ADR/ADR003-use-govuk-signon.md

```diff
@@ -0,0 +1,27 @@
+# ADR003: Use GOV.UK Signon
+
+Date: 2022-02-18
+
+## Status
+
+Proposed
+
+## Context
+
+GOV.UK Forms needs a way of authenticating users who will create and manage forms.
+
+It would be preferably to make use of an existing authentication system rather than create a new one. To explore the options, 2 prototypes were created to better understand the effort to onboard and the account creation process.
```

Comment:
We should write all our ADRs in simple language .When this is edited, could you switch it all to first person, rather than the mix of first and third that we currently have?

For example, rewrite _"To explore the options, 2 prototypes were created..."_ to _"We created 2 prototypes to explore the options..."_

Third person is archaic and inevitably complicates what you are trying to say. Be direct: If we did it, say that. :+1:

Reply:
Isn't `To explore the options, 2 prototypes were created..` technically the passive voice and not necessarily third person?

Either way we should avoid the passive voice too as active is much clearer

Reply:
Passive voice! Yes! That's what I meant. It's even worse than third person, because you're not even NAMING the person who did the action :sweat_smile:

Reply:
Agree that it would make sense to use the active voice here (even though I habitually use the passive voice all the time)